lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +45,28 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_list(phrases, str, False) or not check_list(stop_words, str, False):
+        return None
+    candidate_keyword_phrases = []
+    for phrase in phrases:
+        phrase = phrase.lower()
+        phrases1 = phrase.split()
```

Comment:
` phrases1 ` consider naming variables in a more comprehensive way

Reply:
i also don't really like names ` first_keywords_candidate` and `second_keywords_candidate`, you can turn it to tuple right before appending to the list